cmake/BoBRobotics.cmake

```diff
@@ -55,8 +77,8 @@ function(make_base_target)
         include(CheckCXXCompilerFlag)
         foreach(FLAG IN LISTS DEFAULT_COMPILER_FLAGS)
             check_cxx_compiler_flag(${FLAG} COMPILER_SUPPORTS_${FLAG})
-            if(${VARNAME})
-                target_compile_options(bob_base INTERFACE -${FLAG})
+            if(COMPILER_SUPPORTS_${FLAG})
```

Comment:
1. This is a cool mechanism
2. Whoops :smile:

Reply:
Yeah :laughing:. Although this part only caused the warnings to be turned off (which is also not ideal). The main problem here was that CMake didn't bother applying the default build flags to the targets for some reason.